File: Problem Statements/Swapping 2 Numbers/w3V/swap2NumW.java
import java.util.Scanner;
public class swap2NumW{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter 1st number : ");
        int a=sc.nextInt();
        System.out.print("Enter 2nd number : ");
        int b=sc.nextInt();
        sc.close();
        int c=a;
        a=b;
        b=c;
        System.out.println("1st number : "+a);
        System.out.println("2nd number : "+b);
    }
}

File: Problem Statements/Swapping 2 Numbers/w3V/swap2NumW.cpp
#include<iostream>
using namespace std;
void main(){
    int a,b,c;
    cout<<"Enter 1st number : ";
    cin>>a;
    cout<<"Enter 2nd number : ";
    cin>>b;
    c=a;
    a=b;
    b=c;
    cout<<"1st number : "<<a<<endl;
    cout<<"2nd number : "<<b;
}